src/concurrently.js

```diff
@@ -60,6 +61,15 @@ module.exports = (commands, options) => {
         maybeRunMore(commandsLeft);
     }
 
+    if (options.logger) {
+        const outputWriter = new OutputWriter({
+            outputStream: options.outputStream,
+            group: options.group,
+            commands,
+        });
+        options.logger.observable.subscribe(({command, text}) => outputWriter.write(command, text));
```

Comment:
I initially had this actually! I ended up removing it because then it made look `logger` look like an `Observable`, which I thought might be confusing (cause then folks might expect it to have other `Observable` methods/props). Then I thought maybe I should make `Logger extends Observable`, and then I decided I'd just KISS and use composition :P